### ISSUE TITLE
Increase font size on the Navigation/Top bar

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeUKGenealogyWP#37]

**Comment by carcarvroom** [comment]
Increased top navigation font from 13px to 16px

**Comment by DeniseColbert** [comment]
Thanks Carly, definitely getting closer. 

This relates to the standardisation story as well, but the look and feel of this website should echo what we have on our (newer) project websites (i.e. www.freereg.org.uk & www.freecen.org.uk are live sites, FreeBMD is being redeveloped at test2.freebmd.org.uk - username and password to access this site are both 'test'). @AKHILP96 could you take a look and create a design/spec that blends with our other sites?

**Comment by akhilpokle** [comment]
@DeniseColbert by design spec you mean a design spec that's uniform across freereg, freecen and freebmd ?

**Comment by DeniseColbert** [comment]
Some elements should echo the other websites, for consistency, but the spec
will be specific to this website. I guess we need to think about what
things can we emulate from our project sites, and what things are unique to
this website?

**Comment by akhilpokle** [comment]
So common elements like navigation, footer, etc to have uniform design across the three websites.

**Comment by DeniseColbert** [comment]
I'm not explaining this very well, and not in the office now until Monday,
so will ponder over the weekend and reply then. Apologies, Akhil.

**Comment by akhilpokle** [comment]
@DeniseColbert No Problem
@carcarvroom if you're planning to work on the website, then pick anything apart from homepage and donation page. If possible take a look at the news page design, we need to figure out a way to implement and use Wordpress cms with the news page.

**Comment by DeniseColbert** [comment]
@PrabxDesign to take a look and come up with ideas ahead of UX meet.

**Comment by johnshaw1000** [comment]
Is there anything left to do on this?
The "menu" on FreeUkGen is quite different to the other sites, I assume this is deliberate - if not I can look into this.

**Comment by DeniseColbert** [comment]
The text should look just like FreeCEN/REG (except where they are using all caps: we want title case) whilst keeping the overall design of our nav bar/menu area

**Comment by johnshaw1000** [comment]
The font family and the font size are the same on FreeUKGen as FreeCEN/REG (18px Source Sans Pro).

**Comment by DeniseColbert** [comment]
Thanks! Can the font weight be gleaned via dev tools @johnshaw1000 ?

**Comment by johnshaw1000** [comment]
Yes, I compared via dev tools and they're the same (Source Sams Pro 18pt if memory serves correctly).
